content/doc/book/system-administration/reverse-proxy-configuration-with-jenkins/reverse-proxy-configuration-apache.adoc

```diff
@@ -317,6 +318,163 @@ The _ProxyRequests Off_ prevents Apache from functioning as a forward
 proxy server (except for _ProxyPass_), it is advised to include it
 unless the server should function as a proxy.
 
+== mod_proxy_unix_sockets
+
+Apache can be configured to proxy requests to Jenkins over a Unix domain socket instead of a TCP port. 
+This approach enhances security by eliminating network exposure of Jenkins and is particularly useful 
+when Jenkins and Apache run on the same host.
+
+To avoid repetitive use of `sudo` in the commands below, you can switch to a root shell first by running:
```

Comment:
```suggestion
To avoid repetitive use of `sudo` in the commands below, switch to a root shell first by running:
```
Suggestion to make the wording more active/direct.